storybook/src/admin/Button.stories.tsx

```diff
@@ -75,14 +75,210 @@ export const Default = {
 export const AllVariants = {
     render: () => {
         return (
-            <Stack direction="row" spacing={2}>
-                <Button variant="primary">Primary</Button>
-                <Button variant="secondary">Secondary</Button>
-                <Button variant="outlined">Outlined</Button>
-                <Button variant="destructive">Destructive</Button>
-                <Button variant="success">Success</Button>
-                <Button variant="textLight">Text Light</Button>
-                <Button variant="textDark">Text Dark</Button>
+            <Stack spacing={10}>
+                <Stack direction="row" spacing={8}>
+                    <Button variant="primary">Button</Button>
+                    <Button variant="primary" startIcon={<ArrowRight />}>
+                        Button
+                    </Button>
+                    <Button variant="primary" endIcon={<ArrowRight />}>
+                        Button
+                    </Button>
+                    <Button variant="primary" endIcon={<Chip label="5" />}>
+                        Button
+                    </Button>
+                    <Button variant="primary" startIcon={<ArrowRight />} endIcon={<Chip label="5" />}>
+                        Button
+                    </Button>
+                    <Button
+                        variant="primary"
+                        endIcon={
+                            <>
+                                <ArrowRight />
+                                <Chip label="5" />
+                            </>
+                        }
+                    >
+                        Button
+                    </Button>
+                    <Button variant="primary">
+                        <ArrowRight />
+                    </Button>
```

Comment:
`primary` is already the default `variant`, so setting the `variant` here is not necessary.